pkg/util/network.go

```diff
@@ -30,9 +32,20 @@ func DownloadFile(filepath string, url string) (err error) {
 	if resp.StatusCode != http.StatusOK {
 		return fmt.Errorf("download link %s returned wrong status code: got %v want %v", url, resp.StatusCode, http.StatusOK)
 	}
+	reader := resp.Body
+	if progressBar {
```

Comment:
indeed! I felt like a progress bar was a must, as some archives might be quite large, and offering no visual feedback during a 5 gig download seems like a non-starter.